Constants/Consts.py

```diff
@@ -43,6 +43,14 @@
 degree_freq = 1
 # the radius around the starting position of the robot that should have no obstacles
 tol = int(robot_radius/tile_size + (robot_radius / tile_size) * bloat_factor)
+# left and right motor speeds when moving straight (both get this speed)
+motor_speed = (0.25, 0.25)
+# left and right motor speeds when rotating left in place
+rotation_left = (-0.15, 0.15)
+# left and right motor speeds when rotating right in place
+rotation_right = (0.15, -0.15)
+
```

Comment:
Were these the values for left and right speeds determined from actual testing?

Reply:
Yep, the ECE's determined that those were the motor speeds they wanted to use!